src/game/g_cmds.c

```diff
@@ -2088,6 +2090,23 @@ void Cmd_Load_f(gentity_t *ent) {
 			// Nico, inform client the need to change weapon
 			trap_SendServerCommand(ent - g_entities, va("weaponUpdate %d", pos->weapon));
 		}
+				// enable in vet double load click under 16000 clockunits reset time
+		// clockunits 32bit->1 000 000 clockunits = 1 second
+		now = clock();
```

Comment:
oh actually, you should use level.time as you are in the qagame module and don't have access to cg variables